content/eu-es/howitworks.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Hasi Zaitez
+slug: getting-started
+top_graphic: 3
+aliases : [/howitworks]
```

Comment:
The "aliases:" field in the Hugo header should not be copied. This leads to inconsistent redirects.

Reply:
@mikelsr This is a blocking point, to you think you can fix it?